unittests.py

```diff
@@ -96,6 +96,12 @@ def testBackslashBeforeQuote(self):
         self._genericTest(r'/Fo"C:\out dir\\" /nologo', [r'/FoC:\out dir' '\\', r'/nologo'])
         self._genericTest(r'/c /Fo"C:\out dir\\" /nologo', ['/c', r'/FoC:\out dir' '\\', r'/nologo'])
 
+    def testDoubleQuotedWithSpace(self):
+        self._genericTest(r'"/Fo"some thing\main.obj""', [r'/Fosome thing\main.obj'])
```

Comment:
I think this should actually expand to two arguments,

1. `/Fosome`
2. `thing\main.obj`.